index.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference path='typings/tsd.d.ts' />
+/// <reference path='typings/definitions.d.ts' />
```

Comment:
Well, take an example if we are gonna use `restify.UnsupportedMediaTypeError ` in `index.ts`. `index.ts` has an `import restify = require('restify')`, and it only takes typing from `tsd`, I would guess `tsc` will say can't find. Can you give it a shot in `index.ts`, like in line **101**, substitue `InternalError` with `UnsupportedMediaTypeError`, would it compile?